firebase-vscode/src/core/project.ts

```diff
@@ -67,6 +124,6 @@ async function promptUserForProject(projects: FirebaseProjectMetadata[]) {
     description: p.displayName,
   }));
 
-  const projectId = await vscode.window.showQuickPick(items);
-  return projectId;
+  const item = await vscode.window.showQuickPick(items);
```

Comment:
Why this change? Can we add the return type to eht func

Reply:
I'm using label/description now, instead of just a string.